src/frontend/state/GlobalState.tsx

```diff
@@ -143,6 +150,21 @@ class GlobalState extends PureComponent<Props> {
 
     return games
   }
+
+  loadZoomLibrary = (): Array<GameInfo> => {
+    const games = zoomLibraryStore.get('games', [])
+    const installedGames = zoomInstalledGamesStore.get('installed', [])
+    for (const igame in games) {
+      for (const installedGame of installedGames) {
+        if (installedGame.appName === games[igame].app_name) {
+          games[igame].install = installedGame
+          games[igame].is_installed = true
+        }
+      }
+    }
```

Comment:
There's a `break` missing here, meaning this will currently take much longer to run (depending on your library size of course). We can also make this a bit more readable with `Array.find`
```suggestion
    for (const game of games) {
      const igame = installedGames.find((igame) => igame.appName === game.app_name)
      if (igame) {
        game.install = igame
        game.is_installed = true
      }
    }
```